### PR TITLE
Image v2 working

### PR DESCRIPTION
**Pull request by LucileNaegele**
fibsemImage over

### CODE REVIEW

File: fibsem/acquire.py [hunk 1]
```diff
@@ -147,6 +148,27 @@ def new_image(
     
     return image
 
+def new_fibsemImage(
```
Comment (patrickcleeve2) [inline fibsem/acquire.py]: Please remove all the changes in acquire. @LucileNaegele

File: tests/test_image.py [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+def test_saving_image():
+    """Test saving FibsemImage data to file"""
+
+    array1 = np.uint8(255 * np.random.rand(32, 32))
+    img = fb.FibsemImage(array1)
+    img.save("test.tif")
+    with tff.TiffFile("test.tif") as tiff_image:
+        data = tiff_image.asarray()
+    assert np.array_equal(array1, data)
+    assert img.data.shape[0] == 32
```
Comment (patrickcleeve2) [inline tests/test_image.py]:
```suggestion
    assert img.data.shape == array1.shape
```

File: fibsem/fibsemImage.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
+import numpy as np
```
Comment (patrickcleeve2) [inline fibsem/fibsemImage.py:1]: Please rename file to FibsemImage

File: tests/test_image.py [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+    assert img.data.shape[1] == 32
+    assert img.data.all() >= 0
+    assert img.data.all() <= 255
```
Comment (patrickcleeve2) [inline tests/test_image.py]: the all() tests are not required, we are not enforcing this

File: tests/test_image.py [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+    assert img.data.dtype == np.uint8
+
+
+def test_saving_metadata(img_settings):
+    """Test saving FibsemImage metadata to file.
+
+    Args:
+        img_settings (fixture): fixture returning ImageSettings object
+    """
+    array1 = np.uint8(np.zeros((256, 128)))
```
Comment (patrickcleeve2) [inline tests/test_image.py]:
```suggestion
array1 = np.zeros((256, 128)), dtype=np.uint8)
```

**Review by patrickcleeve2 — CHANGES_REQUESTED** [review]

**Review by LucileNaegele — COMMENTED** [review]
made all the changes/approved Patrick's edits

**Review by patrickcleeve2 — APPROVED** [review]

**Pull request merged**